utils/config.go

```diff
@@ -355,6 +355,7 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["SiteURL"] = strings.TrimRight(*c.ServiceSettings.SiteURL, "/")
 	props["WebsocketURL"] = strings.TrimRight(*c.ServiceSettings.WebsocketURL, "/")
 	props["SiteName"] = c.TeamSettings.SiteName
+	props["EnableAPIv3"] = strconv.FormatBool(*c.ServiceSettings.EnableAPIv3)
```

Comment:
Not sure we want to expose this setting. We don't really want to encourage people to write code that depends on this because we are going to remove the v3 apis soon and any server that supports this setting supports the v4 apis.

Reply:
Ah never mind. I see what this is for.